lib/screens/v2/candidate_generator/triptych.ex

```diff
@@ -20,12 +20,13 @@ defmodule Screens.V2.CandidateGenerator.Triptych do
   @impl CandidateGenerator
   def candidate_instances(
         config,
-        crowding_widget_instances_fn \\ &Widgets.TrainCrowding.crowding_widget_instances/1,
+        opts,
+        crowding_widget_instances_fn \\ &Widgets.TrainCrowding.crowding_widget_instances/2,
         evergreen_content_instances_fn \\ &Widgets.Evergreen.evergreen_content_instances/1,
         local_evergreen_set_instances_fn \\ &Widgets.LocalEvergreenSet.local_evergreen_set_instances/1
       ) do
     [
-      fn -> crowding_widget_instances_fn.(config) end,
+      fn -> crowding_widget_instances_fn.(config, opts) end,
```

Comment:
From our huddle conversation, I suggested this so the train_crowding widget only has the stuff it needs
```suggestion
      fn -> crowding_widget_instances_fn.(config, opts[:logging_options]) end,
```

Reply:
And then the associated changes are needed in the train_crowding candidate_generator

Reply:
Ahh I see. I misunderstood. Will make that change now.